students/westobrien/Lesson07/html_render.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+
+# This is the framework for the base class
+class Element(object):
+
+    tag = "html"
+
+    indent = "    "
+
+    def __init__(self, content=None, **kwargs):
+        if content is None:
+            self.contents = []
+        else:
+            self.contents = [content]
+        self.attributes = []
+        for item in kwargs.items():
+            self.attributes.append(item)
```

Comment:
you can do the same thing with `list(d.items())` although there is no need to convert to a list, you can just use it as a dictionary.